resources/js/components/Button.tsx

```diff
@@ -0,0 +1,12 @@
+export const Button = ({ text }: { text: string }) => {
+  return (
+    <div className="flex justify-center">
+      <button
+        className="rounded-full bg-red-600 px-16 py-2 text-white shadow-lg"
+        type="submit"
```

Comment:
Sometimes you may not want this to be a submit, you could add an optional prop that is "submit" by default, that way you could change it to type "button" as needed